pylint_odoo/checkers/modules_odoo.py

```diff
@@ -128,6 +128,11 @@
         'xml-attribute-translatable',
         settings.DESC_DFLT
     ),
+    'W%d39' % settings.BASE_OMODULE_ID: (
+        '%s Deprecated <data> xml node indide <odoo> xml node',
```

Comment:
@moylop260 
The `%s` in the check name is equal to https://github.com/Vauxoo/pylint-odoo/blob/master/pylint_odoo/checkers/modules_odoo.py#L48 

I try to delete the `%s` in the name then the script show the following error:

![image](https://user-images.githubusercontent.com/1387970/26842462-ec09141c-4aba-11e7-9e7e-27b1bd722328.png)